tests/manage/pv_services/space_reclaim/test_rbd_space_reclaim_block_pvc.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    polarion_id,
+)
+from ocs_ci.ocs.exceptions import (
+    TimeoutExpiredError,
+    UnexpectedBehaviour,
+)
+from ocs_ci.helpers.helpers import check_rbd_image_used_size
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.10")
+class TestRbdSpaceReclaim(ManageTest):
+    """
+    Tests to verify RBD space reclamation
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, project_factory, storageclass_factory, create_pvcs_and_pods):
+        """
+        Create PVCs and pods
+
+        """
+        self.pool_replica = 3
+        pvc_size_gi = 25
+        self.sc_obj = storageclass_factory(
```

Comment:
Please use the default storage class if new storage class is not mandatory. This will ensure the test coverage using the default storage class. There are other reclaim space tier1 test cases which use new storageclass